README.md

```diff
@@ -124,6 +124,8 @@ NUMBER_OF_NEW_ACTIVITIES_TO_PROCESS_PER_IMPORT=250
 # The default schedule runs once a day at 04:05. If you do not know what cron expressions are, please leave this unchanged
 # Make sure you don't run the imports too much to avoid hitting the Strava API rate limit. Once a day should be enough.
 IMPORT_AND_BUILD_SCHEDULE="5 4 * * *"
+# Set the timezone used for the schedule
+TZ=Etc/GMT
```

Comment:
Q: do we need this timezone param? If so, can we rename it be more readable?

Reply:
Yes, this is the ENV  used by LSIO to set the correct timezone inside the container. I'm trying to find a reference to it in their repo but I'm 100% sure it has to be named this. Also fairly confident this is the commonly used name for timezone on other projects.

Reply:
No it's ok :).
Can we have a link in the docs with a list of valid timezones?

Reply:
In the docs as in a comment in the readme? Or in the wiki?

Reply:
Maybe both. As a comment in the .env file above the variable (just like I did for sport types) and in the wiki